osu.Game.Rulesets.Mania/Objects/Drawables/DrawableManiaHitObject.cs

```diff
@@ -22,6 +22,10 @@ public abstract class DrawableManiaHitObject : DrawableHitObject<ManiaHitObject>
 
         protected readonly IBindable<ScrollingDirection> Direction = new Bindable<ScrollingDirection>();
 
+        // Leaving the default (10s) makes hitobjects not appear, as this offset is used for the initial state transforms.
+        // Calculated as DrawableManiaRuleset.MAX_TIME_RANGE + some additional allowance for velocity < 1.
+        protected override double InitialLifetimeOffset => 30000;
```

Comment:
fwiw this did not fix anything for me when testing locally with maximum scroll speed selected.

Reply:
Can you describe what issue you're seeing? This is intended to help with *minimum* scroll speed, not maximum.

Reply:
I wouldn’t have been able to make the screenshot in the description without this, so I doubt that. Are you testing on a different/more extreme beatmap or something?

Reply:
I was specifically testing the same map that I linked in the issue, but I've seen the same thing on any map using any skin.
Here's what I'm seeing: 

https://user-images.githubusercontent.com/35152647/125959941-95cc0a82-de22-4d5a-9126-43a084392a3a.mp4

Any other settings that could affect this?

Reply:
Ah, I see what's going on. It looks like the one that actually needs to change is the lifetime entry's offset, because that determines when the hitobject first gets applied. But that also has an actual overhead if applied to the same value, so I'm less keen to change it.

@ekrctb You had changes before that calculated the lifetime without requiring DHO sizing and all that, do you still think that's going to happen sometime? `InitialLifetimeOffset` is quite flaky...

Reply:
@smoogipoo I don't have immediate plan to address that as I couldn't figure out how to do change in a backward compatible manner.